dev-python/uri_template/uri_template-1.1.0.ebuild

```
@@ -0,0 +1,21 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( pypy3 python3_{8..10} )
+inherit distutils-r1
```

Please use here [PEP517 mode](https://projects.gentoo.org/python/guide/distutils.html#basic-use-pep-517-mode)
```suggestion
DISTUTILS_USE_PEP517=setuptools
PYTHON_COMPAT=( pypy3 python3_{8..10} )
inherit distutils-r1
```